libmscore/pitchspelling.h

```diff
@@ -25,6 +25,7 @@ enum class Key;
 const int   INVALID_PITCH      = -1;
 
 // a list of tpc's, with legal ranges, not really an enum, so no way to cnvert into a class
+// DO NOT rearrange this enum, otherwise the logic in scale.cpp will break!!!
 enum Tpc : signed char {
```

Comment:
I can remove it. Although it would be interesting to test if anything would break if I swap BB notes with SS.